webservices/common/models/itemized.py

```diff
@@ -118,11 +121,36 @@ class ScheduleA(BaseItemized):
     contributor_aggregate_ytd = db.Column('contb_aggregate_ytd', db.Numeric(30, 2))
 
     # Related candidate info
+    candidate_id = db.Column('cand_id', db.String, doc=docs.CANDIDATE_ID)
+    candidate_name = db.Column('cand_nm', db.String, doc=docs.CANDIDATE_NAME)
+    candidate_first_name = db.Column('cand_nm_first', db.String)
+    candidate_last_name = db.Column('cand_nm_last', db.String)
+    candidate_middle_name = db.Column('cand_m_nm', db.String)
+    candidate_prefix = db.Column('cand_prefix', db.String)
+    candidate_suffix = db.Column('cand_suffix', db.String)
     candidate_office = db.Column('cand_office', db.String)
-    candidate_office_description = db.Column('cand_office_desc', db.String)
-    candidate_office_district = db.Column('cand_office_district', db.String)
+    candidate_office_full = db.Column('cand_office_desc', db.String)
+    candidate_office_state = db.Column('cand_office_st', db.String)
+    candidate_office_state_full = db.Column('cand_office_st_desc', db.String)
+    candidate_office_district = db.Column('cand_office_district', db.String, doc=docs.DISTRICT)
+
+    # Conduit info
+    conduit_committee_id = db.Column('conduit_cmte_id', db.String)
+    conduit_committee_name = db.Column('conduit_cmte_nm', db.String)
+    conduit_committee_street1 = db.Column('conduit_cmte_st1', db.String)
+    conduit_committee_street2 = db.Column('conduit_cmte_st2', db.String)
+    conduit_committee_city = db.Column('conduit_cmte_city', db.String)
+    conduit_committee_state = db.Column('conduit_cmte_st', db.String)
+    conduit_committee_zip = db.Column('conduit_cmte_zip', db.Integer)
+
+    donor_committee_name = db.Column('donor_cmte_nm', db.String)
+    national_committee_nonfederal_account = db.Column('national_cmte_nonfed_acct', db.String)
+    cycle = db.Column('election_cycle', db.Integer)
+    timestamp = db.Column('timestamp', db.DateTime)
```

Comment:
Ah nice, thanks @LindsayYoung!  That makes total sense then with the field changes.

And yes, we'll have to check what the downloads look like (and the API responses themselves).  It would be a good opportunity for usability testing on these features! :-)